lib/config/presets/internal/config.ts

```diff
@@ -15,6 +15,20 @@ export const presets: Record<string, Preset> = {
       'workarounds:all',
     ],
   },
+  'best-practices': {
```

Comment:
Merging this PR will close the linked issue. But that issue needs more work from @HonkingGoose after merging. Can you edit the top comment of this PR , like this:

```diff
+ Related issue #16100 
- closes https://github.com/renovatebot/renovate/issues/16100
```

After you've done that a maintainer probably needs to unlink this PR from the issue in the PR sidebar.